src/seer/automation/autofix/steps/steps.py

```diff
@@ -1,7 +1,7 @@
 import os
 import threading
 import time
-from typing import Any
+from typing import Any, Optional
```

Comment:
is this import used?
```suggestion
from typing import Any
```